diagnostic.md

```diff
@@ -93,4 +89,4 @@ var teamMember = {
 
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+Jason, Weeks, exercise 
```

Comment:
it would be 
teamMember['hobby'] = 'exercise';

teamMember['name'] = {
  given: 'Jason',
  surname: 'Weeks'
};

teamMember['name']['surname'] = 'Weeks';